draft-ietf-oauth-identity-chaining.md

```diff
@@ -229,9 +255,23 @@ The authorization server MUST validate the JWT authorization grant as specified
 
 The authorization server responds with an access token as described in section 5.1 of {{RFC6749}}.
 
+The following processing rules applies:
+
+* If the JWT authorization grant in the Access Token Request contains a "requested_cnf" claim:
+  * The authorization server in Domain B MUST verify the proof of possession for the private key corresponding to the top level "cnf" claim.
+  * The authorization server MUST include the "cnf" value from the "requested_cnf" claim in the returned token.
+* If Domain B does not support sender constrained tokens and the rAccess Token Request is sender constrained, AS-B MAY issue access tokens that are not sender constrained.
+* If Domain B requires constrained tokens and the Token Exchange Rquest is not sender constrained, AS-B MUST reject the request and return an error response per Section 5.2 of {{RFC6749}} with invalid_pop as the value of the error parameter.
```

Comment:
would need to request registration of a new error code...

Reply:
and we should really try to be consistent with how protocol elements are quoted or ticked or what. Or at least try not to more inconsistent :)